### PR TITLE
Code style improvements

### PR DESCRIPTION
**Pull request by felixrindt**
Just some stuff I noticed while looking at #1396 :)

### CODE REVIEW

File: src/pretix/base/payment.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def settings_form_fields(self) -> dict:
 
         We suggest that you return an ``OrderedDict`` object instead of a dictionary
         and make use of the default implementation. Your implementation could look
-        like this::
```
Comment (rixx) [inline src/pretix/base/payment.py]: The "::" is intentional, as it signals in RST that a code block is about to begin. You can see the output here: https://docs.pretix.eu/en/latest/development/api/payment.html#pretix.base.payment.pretix.base.payment.BasePaymentProvider.BasePaymentProvider.settings_form_fields

File: src/pretix/plugins/banktransfer/payment.py [hunk 2]
```diff
@@ -116,16 +121,7 @@ def test_mode_message(self):
...
-            list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()) + [
-                ('omit_hyphen', forms.BooleanField(
-                    label=_('Do not include a hypen in the payment reference.'),
-                    help_text=_('This is required in some countries.'),
-                    required=False
-                )),
-
-            ]
-        )
+        d = OrderedDict(list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()))
```
Comment (raphaelm) [inline src/pretix/plugins/banktransfer/payment.py]: This is also on purpose. This hyphen thing is a very specific option, only really a problem in Finland. With your change, we would start asking it to every user who goes through the quick event setup.

**Comment by felixrindt** [comment]
Haha ok sorry for the interruption then and thanks for explaining ;)

File: src/pretix/plugins/banktransfer/payment.py (hunk 2 above)
Comment (felixrindt) [reply src/pretix/plugins/banktransfer/payment.py]: I searched for usages of `BankTransfer.form_fields()`. In https://github.com/pretix/pretix/blob/76e75bef65a9543594ac418e0ebc16f61a7f9a0e/src/pretix/control/forms/event.py#L1495

the fields shown in the quick setup are hardcoded, so I thought that wouldn't be a problem.

File: src/pretix/base/payment.py (hunk 1 above)
Comment (felixrindt) [reply src/pretix/base/payment.py]: Thanks for explaining. I removed that change.

File: src/pretix/plugins/banktransfer/payment.py (hunk 2 above)
Comment (raphaelm) [reply src/pretix/plugins/banktransfer/payment.py]: Agreed

**Pull request merged**